src/libostree/ostree-sysroot-deploy.c

```diff
@@ -1641,15 +1641,15 @@ full_system_sync (OstreeSysroot *self, SyncStats *out_stats, GCancellable *cance
                   GError **error)
 {
   GLNX_AUTO_PREFIX_ERROR ("Full sync", error);
-  ot_journal_print (LOG_INFO, "Starting syncfs() for /ostree");
+  ot_journal_print (LOG_INFO, "Starting syncfs() for physical root");
   guint64 start_msec = g_get_monotonic_time () / 1000;
-  glnx_autofd int ostree_dfd = -1;
-  if (!glnx_opendirat (self->sysroot_fd, "ostree", TRUE, &ostree_dfd, error))
```

Comment:
We use `ostree_dfd` in `create_new_bootlinks` and `swap_bootlinks`, that's why I went with it initially